vse-quick-start/1-build-image-gallery.md

```diff
@@ -72,23 +72,24 @@ $("#images").kendoMobileListView({
 
 ### Step 3. Deploy your app using the companion app
 
-The AppBuilder companion app makes it easy to test your app on real devices, without the need to manage SDKs or deal with complex provisioning options.
+The Telerik Platform Companion app is a dashboard app that lists the applications on your account and lets you scan QR codes. The Companion app utilizes player apps to run your applications in a sandbox environment. As such, you’ll use the Companion app to load apps on your account, and the player app to run your apps on real devices, without the need to manage SDKs or deal with complex provisioning options.
 
 <hr data-action="start" />
 
 #### Action
 
 * **a.** Get your iOS, Android, or Windows Phone device out.
-* **b**. Download and install the AppBuilder companion app from your device's app store.
+* **b**. Download and install the Telerik Platform (AppBuilder for Windows Phone) Companion app from your device's app store — i.e. the App Store for iOS users, Google Play for Android users, or the Windows Store for Windows Phone users (search for **Platform Companion** for iOS/Android and **AppBuilder** for Windows Phone or use the links provided below).
 
 [![iOS app store](images/app-store-icon.png)](https://itunes.apple.com/us/app/telerik-appbuilder/id527547398?mt=8)
 [![Google Play](images/google-play-icon.png)](https://play.google.com/store/apps/details?id=com.telerik.AppBuilder&hl=en)
 [![Windows Phone Store](images/windows-phone-store-icon.png)](https://www.windowsphone.com/en-us/store/app/appbuilder/0171d46b-b5f2-43d9-a36b-0a78c9692aab?signin=true)
 
-* **c**. Open the AppBuilder app on your device, and then use a two-finger swipe to reveal the companion app's menu. 
+* **c**. **iOS or Android:** Open the Telerik Platform Companion app on your device, and tap on **Help & About**. In the newly opened page, locate the Cordova section and select **Install Dev Application**. You will be redirected to your device's app store, where you should click **Install**. This will download and install the Cordova player app, which the Telerik Platform Companion app will use to run your applications.
+* **c**. **Windows Phone:** Open the AppBuilder Companion app on your device, and then use a two-finger swipe to reveal the Companion app's menu.
```

Comment:
Dupcate c action e.g. `* **c**. **Windows Phone:**` should be `* **d**. **Windows Phone:**` and the latter should be `* **e**`.